components/ModalView/ModalViewGene.vue

```diff
@@ -148,7 +148,9 @@
             this.data = data;
           })
           .catch(function (error) {
-            console.log(error);
+            $store.commit('set_alert_modal', {
+              msg: 'Failed to get data in Modal View Gene',
+            });
```

Comment:
この間のレビューで見落としましたが、この場合は必ず`this.`を指定する必要があります。`Store`を呼ぶメソッドもVue component内にしか存在していないからです。
また、...mapMutationsでメソッドを一回定義した方が良いかもしれません。
つまり、エラーのCatchが以下となります：
```suggestion
            this.setAlertModal({msg: 'Failed to get data in Modal View Gene'});
```

`setAlertModal`のメソッドを`Methods`で設定します：
```    Javascript
methods: {
      ...mapMutations({
        setGeneModal: 'set_gene_modal',
        setAlertModal: 'set_alert_modal',
      }),
    },
```

気づくのが遅くて申し訳ありません！

Reply:
詳細な説明ありがとうございます